watershed-android/watershed/app/src/main/java/com/blueprint/watershed/Sites/SiteListFragment.java

```diff
@@ -87,14 +79,22 @@ public void onRefresh() {
                 getSitesRequest();
             }
         });
+
+        if (mSites == null) {
+            hideList();
+            mNoSiteLayout.setRefreshing(true);
+            getSitesRequest();
+        }
+
+        mAdapter = new SiteListAdapter(mParentActivity, R.layout.site_list_row, getSites());
+        mSiteListView.setAdapter(mAdapter);
     }
 
     @Override
     public void onResume() {
         super.onResume();
         mParentActivity.setMenuAction(true);
-        mSwipeLayout.setRefreshing(true);
-        getSitesRequest();
+//        getSitesRequest();
```

Comment:
I think we can safely delete this entirely. Since we're no longer refreshing the list every time the user opens the app.